eng/Version.Details.xml

```diff
@@ -21,9 +101,17 @@
       <Uri>https://github.com/dotnet/coreclr</Uri>
       <Sha>20904a2629729e33531f30d87ae04b285cb34ee7</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Private.WindowsDesktop.App" Version="3.0.0-preview6-27702-36">
-      <Uri>https://dev.azure.com/devdiv/DevDiv/_git/DotNet-Trusted</Uri>
-      <Sha>45ef76ae65decfab78855206936a12b00d9dd87a</Sha>
+    <Dependency Name="Microsoft.Private.Winforms" Version="4.8.0-preview6.19251.3">
+      <Uri>https://github.com/dotnet/winforms</Uri>
+      <Sha>084316b6d123706f5e2e075f5bdf2e08f0a69f9a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Wpf.GitHub" Version="4.8.0-preview6.19251.2">
+      <Uri>https://github.com/dotnet/wpf</Uri>
+      <Sha>efd2cd96ea72393ff3dd9174a2d1fd0262fa0dc9</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Wpf.DncEng" Version="4.8.0-preview6.19251.6">
```

Comment:
I think I'd want this to say `CoherentParentDependency="Microsoft.DotNet.Wpf.GitHub"`

/cc @mmitche, @zsd4yr

Reply:
Sounds reasonable to me. Any reason it isn't that way in DN-T?

Reply:
We were talking about adding these near the end of P4 but never got around to it...we should add these to wpf repos as well. 

WinForms should have a dependency on netcore.platforms, I think. (right, @zsd4yr?)

Reply:
> I think I'd want this to say `CoherentParentDependency="Microsoft.DotNet.Wpf.GitHub"`
> 
> /cc @mmitche, @zsd4yr

I think you want this to be on the Microsoft.DotNet.Wpf.GitHub dependency, with Microsoft.DotNet.Wpf.DncEng as the coherent parent.

Reply:
I thought we had decided that Winforms would be the coherency parent of wpf.github and then wpf.github would be the parent of wpf.dnceng?

Reply:
Closing/resolving this thread--let's talk about the problem as a whole in the other one. This is confusing. 😛